ddtrace/llmobs/_writer.py

```diff
@@ -263,6 +268,71 @@ def _data(self, events: List[LLMObsEvaluationMetricEvent]) -> Dict[str, Any]:
         return {"data": {"type": "evaluation_metric", "attributes": {"metrics": events}}}
 
 
+class LLMObsExperimentsClient(BaseLLMObsWriter):
```

Comment:
Is this client also responsible for submitting experiments data? If so, should we separate this client into `LLMObsDatasetsClient` to perform the dataset downloads and a `LLMObsExperimentsWriter` to automatically submit experiments data?

Reply:
I'm generalizing experiments here to also include datasets since I don't think it's meaningful to a customer or reader of this code to differentiate

Reply:
Fair point 👍